### ISSUE TITLE
Warning: Each child in an array or iterator should have a unique "key" prop.

### ISSUE THREAD

**Issue by YagoLopez** [nsisodiya/react-json-viewer#16]
I find react-json-viewer very useful for one of my projects, but using this json dynamically generated:

```javascript
[
  {"id":1,"name":"user1","age":11,"admin":1,  
    "posts":[
      {"id":1,"title":"title post 1","content":"content post 1","user_id":1,
        "comments":[
           {"id":1,"author":"author1","date":"date1","post_id":1},
           {"id":2,"author":"author2","date":"date2","post_id":1}]},
     {"id":2,"title":"title post 2","content":"content post 2","user_id":1,
        "comments":[]}]},
  {"id":2,"name":"user2","age":22,"admin":1,"posts":[
      {"id":3,"title":"title post 3","content":"content post 3","user_id":2,
        "comments":[]}]},
  {"id":3,"name":"user3","age":33,"admin":1,"posts":[]}
]
```
React complains:

```javascript
index.js:2177 Warning: Each child in an array or iterator should have a unique "key" prop.

Check the render method of `t`. See https://fb.me/react-warning-keys for more information.
    in tr (created by t)
    in t (created by App)
    in div (created by App)
    in App
```
Is there any way to fix this?
Thanks

**Comment by nsisodiya** [comment]
Issue fixed.  today released 2.0.0 release with major code changes.

**Comment by nsisodiya** [comment]
@YagoLopez Can you try with the latest release?

**Comment by nsisodiya** [comment]
Fixed in 2.0.1
